service/policy/objects.proto

```diff
@@ -105,6 +105,7 @@ enum SubjectMappingOperatorEnum {
   SUBJECT_MAPPING_OPERATOR_ENUM_UNSPECIFIED = 0;
   SUBJECT_MAPPING_OPERATOR_ENUM_IN = 1;
   SUBJECT_MAPPING_OPERATOR_ENUM_NOT_IN = 2;
+  SUBJECT_MAPPING_OPERATOR_ENUM_CONTAINS = 3;
```

Comment:
Could we please add comments to the proto/gencode about how (if I read correctly), contains evaluates a string match against every value in the selected values? It's a little hard to decipher how it's different from "in" from just looking at the proto, but I think it's OR/ANY-evaluation with substring matches.

Reply:
Alternatively we could just look at the first value and document that the contains (and future startsWith/endsWith) only look at one value within the selected values list. Someone could always move it up out of a single Condition into multiple Conditions in an OR ConditionGroup if they need more complex evaluation.

Reply:
And if we do want to only consider one value in a list for `contains/startsWith/endsWith`, we could validate the length of the `external_selector_values` on the create request in policy to enforce that API behavior.